openassessment/templates/openassessmentblock/staff_debug/student_info.html

```diff
@@ -236,6 +236,6 @@ <h3 class="title">{% trans "Example-Based Assessment" %}</h3>
         {% endif %}
     </div>
     {% else %}
-        {% trans "Couldn't find a response for this student." %}
+        {% trans "Couldn't find a response for this learner." %}
```

Comment:
@andy-armstrong Yes, we discourage contractions. I was wrestling with the strange wording in this message. The least of many evils might be to go with passive voice and say:
"A response was not found for this learner."
@dianakhuang Sorry, can we make this change too?